YevheniiaBondarieva/Node.js_Course_Practice
# [HW2] Refactor app to use TypeScript

Requirements
1. Translating the Project to TypeScript:
- Create a copy of your existing Node.js project.
- Translate all server files and configurations to TypeScript.
- Install necessary packages for TypeScript development (e.g., ts-node, @types/express, etc.).
2. Preserve Functionality:
- Ensure all previous functionality, including the health-check endpoint and Swagger documentation, remains operational.
3. Typing:
- Add types to all variables, parameters, and functions in your application, including Express.js routes and Swagger specifications.
- Use types to ensure safety and provide hints during development.
4. Testing:
- Verify that your translated application still functions correctly, including the health-check endpoint and Swagger documentation.
- Use linters and other tools to check for errors.
